app/networkapi/features/admin.py

```diff
@@ -6,7 +6,8 @@
 
 class FeatureAdmin(SortableAdmin):
     sortable_change_list_template = (
-        'shared/adminsortable_change_list_custom.html'
+        'shared/adminsortable_change_list_custom.html',
```

Comment:
THE WORST EVER! This is what initially broke it for me and it wouldn't load any of the adminsortable stuff.